content/settings/README.md

```diff
@@ -0,0 +1,20 @@
+# Content / Settings
+
+### routes.yaml
+
+This is how the default `routes.yaml` file looks like:
+
```

Comment:
I was thinking of adding some more text here with a link to the docs as soon as those are in place.